src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.fiqures.Circle;
+import core.basesyntax.fiqures.IsoscelesTrapezoid;
+import core.basesyntax.fiqures.Rectangle;
+import core.basesyntax.fiqures.RightTriangle;
+import core.basesyntax.fiqures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int maxValue = 4;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(maxValue);
+        Figure rez = new Circle(new Random().nextInt(10));

```

Comment:
lets move 
```suggestion
        Figure figure;
```

lets define some figure in `default` part of swich